domains/nico.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/NicoGaming77/NicoGaming77.github.io",
+  "owner": {
+    "username": "NicoGaming77",
+    "email": "nicodreyer7@icloud.com",
+    "discord": "nicosnico#7674"
+  }
```

Comment:
There should be a comma after each object unless there aren't any more objects.

Reply:
JSON doesn't support trailing commas, so remove the comma after `"discord"` key, I was asking you to put a comma after the owner object key.